templates/core/terraform/network/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  core_services_vnet_subnets            = cidrsubnets(var.address_space, 2, 4, 4, 2, 2)
```

Comment:
Has this been checked as appropriate? /16 is likely too large for our requirements, and not sure about these allocations.

Reply:
No, but I haven't seen any specs on this, so defaulting to something. We can add a story to specify the subnet space @christoferlof